Ascend.java

```diff
@@ -2,6 +2,23 @@
 // and prints them in increasing order.
 public class Ascend {
 	public static void main(String[] args) {
-		// Replace this comment with your code
+		int lim = Integer.parseInt(args[0]);
+		// Get 3 random numbers according to lim 
+		double a = Math.random() * lim;
+		double b = Math.random() * lim;
+		double c = Math.random() * lim;
+		// Calc max value 
+		double max1 = Math.max(a,b); 

```

Comment:
**Optional: instead of**:
double max1 = Math.max(a,b); 
double max2 = Math.max(b,c);
**Use**:
double max = Math.max(Math.max(a,b), c)